crypto/evp/pmeth_gn.c

```diff
@@ -200,7 +200,7 @@ int EVP_PKEY_gen(EVP_PKEY_CTX *ctx, EVP_PKEY **ppkey)
             && evp_keymgmt_util_gen_get_keytype(&ctx->keytype, ctx->keymgmt,
                                                 ctx->op.keymgmt.genctx)
             && (keytype == NULL
-                || strcmp(keytype, ctx->keytype) == 0
+                || strcasecmp(keytype, ctx->keytype) == 0
```

Comment:
normally have to include "e_os.h" for this..

Reply:
Fixed